src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/ReviewConfirmModal/TokensSectionDiff.tsx

```diff
@@ -58,11 +63,16 @@ export function TokensSectionDiff() {
 
   const formattedReservedRate = parseFloat(formatReservedRate(newReservedRate))
 
+  const currentMintRateAfterDiscountRateApplied = deriveNextIssuanceRate({
+    weight: BigNumber.from(0),
```

Comment:
maybe a comment here to say wahts goin on.

maybe even pull this out into a function an reuse it in these 2 spots. Want to contain the confusion if possible